### DISCUSSION TITLE
New Scanner/Monitoring tool based of pi.alert

### DISCUSSION THREAD

**Discussion by Mithias-uk** [tteck/Proxmox#3238] [Ideas]
Any chance you could create a builder script based off this?

https://github.com/jokob-sk/NetAlertX

Its a fork of pi.alert thats maintained but has a few different features to the ones thats offered (mainly different notification options). Would be great if we could build with this as an alternative :-)

**Comment by Mithias-uk** [comment]
Bump!

I see they have an installer available already for Docker but not LXC. There is a "HW Install" method available too which would *probably* suit your scripts but would obviously need to be checked and integrated first - and I've not got the knowledge for that part :-(

Pi.Alert and its various branches always seem to be missing *something* so I'm excited to see about giving this a try if you can find the time @tteck

**Comment by DomJeal** [comment]
You could always add a docker LXC then use the image in the NetAlertX repo.

**Comment by Mithias-uk** [comment]
Yeah I know - trying to avoid virtualisation nesting like that where possible though as then it's another thing to manage (Even if it self runs probably!). Also handy being able to manage everything through the ProxMox GUI

**Comment by nsmichael82** [comment]
Bumping myself - this would be nice to have.

**Comment by Zazarothh** [comment]
This would be sick, as always, thank ya for your time @tteck

**Comment by pmi24** [comment]
I'd love to see this too! Unfortunately, I'm lacking the knowledge to do this properly myself.
